project_stock_product_set/i18n/es.po

```diff
@@ -0,0 +1,129 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* project_task_material_set
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 13.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-10 06:29+0000\n"
+"PO-Revision-Date: 2022-05-10 08:30+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 2.3\n"
+
+#. module: project_stock_product_set
+#: model_terms:ir.ui.view,arch_db:project_stock_product_set.product_set_add_from_task_form_view
+#: model_terms:ir.ui.view,arch_db:project_stock_product_set.view_task_form2
+msgid "Add set"
+msgstr "Añadir conjunto"
+
+#. module: project_stock_product_set
+#: model:ir.actions.act_window,name:project_stock_product_set.act_open_wizard_product_set_add_from_task
+msgid "Add set in poject task"
+msgstr "Añadir conjunto en la tarea"
+
+#. module: project_stock_product_set
+#: model_terms:ir.ui.view,arch_db:project_stock_product_set.product_set_add_from_task_form_view
+msgid "Add set in sale order line"
+msgstr "Agregar conjunto en línea de pedido de venta"
+
+#. module: project_stock_product_set
+#: model_terms:ir.ui.view,arch_db:project_stock_product_set.product_set_add_from_task_form_view
+msgid "Cancel"
+msgstr "Cancelar"
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,field_description:project_stock_product_set.field_product_set_add_from_task__create_uid
+msgid "Created by"
+msgstr "Creado por"
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,field_description:project_stock_product_set.field_product_set_add_from_task__create_date
+msgid "Created on"
+msgstr "Creado el"
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,field_description:project_stock_product_set.field_product_set_add_from_task__partner_id
+msgid "Customer"
+msgstr "Cliente"
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,field_description:project_stock_product_set.field_product_set_add_from_task__display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,help:project_stock_product_set.field_product_set_add_from_task__skip_existing_products
+msgid "Enable this to not add new lines for products already included in SO lines."
+msgstr ""
```

Comment:
Falta

Reply:
Some terms do not need to be translated because they are not used and/or not shown.

Reply:
But for completion reasons, it's better to simply not let them untranslated

Reply:
And other times, we think they are not shown, but they are. Example: fields not shown on views, but shown on advanced search filters.